### PR TITLE
[4.0] Use namespaced classes for layouts [Part 6]

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

Use namespaced classes for:

- `layouts/joomla/html/**`
- `layouts/joomla/sidebars/**`
- `layouts/joomla/system/**`
- `layouts/joomla/tinymce/**`

### Testing Instructions

code review

### CODE REVIEW

File: layouts/joomla/tinymce/togglebutton.php [hunk 1]
```diff
@@ -9,16 +9,19 @@
...
 ?>
 <div class="toggle-editor btn-toolbar float-right clearfix mt-3">
 	<div class="btn-group">
 		<a class="btn btn-secondary" href="#"
 			onclick="tinyMCE.execCommand('mceToggleEditor', false, '<?php echo $name; ?>');return false;"
-			title="<?php echo JText::_('PLG_TINY_BUTTON_TOGGLE_EDITOR'); ?>"
+			title="<?php echo Text::_('PLG_TINY_BUTTON_TOGGLE_EDITOR'); ?>"
 		>
-			<span class="icon-eye" aria-hidden="true"></span> <?php echo JText::_('PLG_TINY_BUTTON_TOGGLE_EDITOR'); ?>
+			<span class="icon-eye" aria-hidden="true"></span>
```
Comment (Quy) [inline layouts/joomla/tinymce/togglebutton.php:23]: Use `fa`?

Comment (C-Lodder) [reply layouts/joomla/tinymce/togglebutton.php:23]: Ok I'll do all that in another PR. Only did a few as I was going along

File: layouts/joomla/tinymce/buttons.php [hunk 2]
```diff
@@ -9,5 +9,8 @@
 
 defined('JPATH_BASE') or die;
 
-JLog::add('The layout joomla.tinymce.buttons is deprecated, use joomla.editors.buttons instead.', JLog::WARNING, 'deprecated');
```
Comment (wilsonge) [inline layouts/joomla/tinymce/buttons.php]: Kill this file

File: layouts/joomla/tinymce/buttons/button.php [hunk 3]
```diff
@@ -9,5 +9,8 @@
 
 defined('JPATH_BASE') or die;
 
-JLog::add('The layout joomla.tinymce.buttons.button is deprecated, use joomla.editors.buttons.button instead.', JLog::WARNING, 'deprecated');
-echo JLayoutHelper::render('joomla.editors.buttons.button', $displayData);
+use Joomla\CMS\Layout\LayoutHelper;
```
Comment (wilsonge) [inline layouts/joomla/tinymce/buttons/button.php]: And this one

**Pull request merged**